### PR TITLE
Feature/menu/adding logs to cli menu advanced flow

### PR DESCRIPTION
**Pull request by Miquilin**
Added the needed logs to CLIMenuAdvancedFlow class

### CODE REVIEW

**Review by ovandononi — CHANGES_REQUESTED** [review]
Obs 1: there are still some commented lines
Obs 2: Unit test are failed
To verify it, add to the end the following lines on src/com/jalasoft/search_files/menu/cli_menu_advanced_flow.py: 
if __name__ == "__main__":
    unittest.main()

{code}
----------------------------------------------------------------------
Ran 116 tests in 0.091s

FAILED (failures=10)
{code}

**Comment by Miquilin** [comment]
I forgot to comment about the code that is commented in some functions, it is due to I am mocking the return data formthe validation menu function in order to have running the advanced menu. This part of the code (menu_validations) was implemented by @Saex05 and was completed yesterday and merged today to the develop.
About the unit test failing , these all are related to the new validations implemented by @Saex05 
I don't know if it possible to do the merge of this code related to the logs, because I will work on these observations in another branch. Do you agree @ovandononi, @Saex05, @jpsandovaln ??

**Review by Saex05 — APPROVED** [review]

File: src/com/jalasoft/search_files/menu/cli_menu_advanced_flow.py [hunk 1]
```diff
@@ -10,176 +11,303 @@
...
+        # if self.validator.has_valid_characters(option):
+        #     logger.debug("The value returned is: %s", True)
+        #     logger.info("Ending the method")
+        #     return True
+        # else:
+        #     logger.debug("The value returned is: %s", False)
+        #     logger.info("Ending the method")
+        #     return False
 
     def advanced_search_for_content_criteria_validation(self, option):
```
Comment (Saex05) [inline src/com/jalasoft/search_files/menu/cli_menu_advanced_flow.py]: this method is not yet implemented for this reason the unit test fails
`def test_advanced_search_for_content_criteria_validation_returns_false_when_option_provided_has_non_valid_chars(self):`

**Comment by Saex05** [comment]
the logs were correctly applied, the problem with the unit tests are because some methods are not yet  implemented and they are needed the latest validations

**Pull request merged**